protected/humhub/modules/space/widgets/views/myMembership.php

```diff
@@ -11,12 +11,10 @@
 
 <div class="panel panel-default panel-my-membership" id="my-membership-panel">
     <?= PanelMenu::widget(['id' => 'space-my-membership-panel']); ?>
-    <div class="panel-heading"><?= Yii::t('SpaceModule.base', '<strong>My Membership Info</strong>'); ?></div>
+    <div class="panel-heading"><?= Yii::t('SpaceModule.base', '<strong>About</strong> my membership'); ?></div>
     <div class="panel-body">
-        <div><b><?= Yii::t('SpaceModule.base', 'Current Role') ?>: </b><?= ucfirst($role) ?></div>
-        <div><b><?= Yii::t('SpaceModule.base', 'Permissions') ?>: </b>
-            <div><?= empty($permissions) ? '-' : implode(", ", $permissions) ?></div>
-        </div>
```

Comment:
As the Permissions part is deleted from the MyMembership view, So we can also remove `getPermissions` function from the MyMembership widget.

Reply:
I have removed the permissions code at first, maybe we will take it up again in a future version.